camel/loaders/github_loader.py

```diff
@@ -0,0 +1,80 @@
+# =========== Copyright 2023 @ CAMEL-AI.org. All Rights Reserved. ===========
+# Licensed under the Apache License, Version 2.0 (the “License”);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an “AS IS” BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =========== Copyright 2023 @ CAMEL-AI.org. All Rights Reserved. ===========
+from github import Auth, Github
+
+
+class GitHubLoaderIssue:
+    def __init__(self, title, body, number, file_path, file_content):
+        self.title = title
+        self.body = body
+        self.number = number
+        self.file_path = file_path
+        self.file_content = file_content
```

Comment:
We may need a better data structure. It does not scale to large issues that related to more than one file.